test/graaflib/algorithm/strongly_connected_components/kosaraju_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include <graaflib/algorithm/strongly_connected_components/kosaraju.h>
+#include <gtest/gtest.h>
+
+namespace graaf::algorithm {
+
+TEST(KosarajuTest, EqualOutputSCCS) {
+  using sccs_t = std::vector<std::vector<vertex_id_t>>;
```

Comment:
Nice using-declaration 👍🏻 Might also be cool to move this to the source file!

Reply:
Awesome - does it make sense to pop it into `types`? Or to make a `types` header for strongly connected components, as I know Tarjan would also benefit from this?

Reply:
The `types.h` file is meamt for project wide types such as `vertex_id_t`. But I like the idea of sharing the typedef with Tarjan'!

What do you think about adding a `strongly_connected_components/common.h`?

Reply:
> The `types.h` file is meamt for project wide types such as `vertex_id_t`. But I like the idea of sharing the typedef with Tarjan'!
> 
> What do you think about adding a `strongly_connected_components/common.h`?

I think that makes a lot of sense, and any file referencing Tarjan / Kosaraju / etc. will have access to `common` because it will be `include`d in those headers. Will give it a go!